.github/workflows/create-github-release.yml

```diff
@@ -74,13 +74,14 @@ jobs:
           path: 'package.json'
           prop_path: 'version'
       - name: Create Github Release
-        uses: actions/create-release@v1
-        env:
-          GITHUB_TOKEN: ${{ secrets.SVC_CLI_BOT_GITHUB_TOKEN }}
+        uses: ncipollo/release-action@v1.13.0
         with:
-          tag_name: ${{ steps.packageVersion.outputs.prop }}
-          release_name: ${{ steps.packageVersion.outputs.prop }}
-          prerelease: ${{ !contains(fromJSON('["latest", "latest-rc", "nightly"]'), needs.validate-channel.outputs.channel) }}
+          name: ${{ steps.packageVersion.outputs.prop }}
+          tag: ${{ steps.packageVersion.outputs.prop }}
+          commit: ${{ github.ref_name }}
```

Comment:
why is `commit` here? docs say that it's used if there's no tag 
https://github.com/ncipollo/release-action#action-inputs